### DISCUSSION TITLE
Manipulating the UI

### DISCUSSION THREAD

**Discussion by exitxhu** [skoruba/IdentityServer4.Admin#890] [Q&A] [answered]
Hi
first, I appreciate the time and effort you put into building this awesome template.
second, I really want to edit the raw HTML behind the ADMIN UI, cuz I need to do some adjustments for the Persian language which is RTL and I like to change how the dashboard looks.

I noticed that all the UI elements actually are inside a nugget package (skoruba UI) and there is no direct access.

how could I achieve that? is there any way?

**Comment by exitxhu** [comment, marked as answer]
i think i found out what i wanted
i only used skoruba via template install, but now i looked deeper in the repo and find the UI files.
cloned the Repo and now i can access what i want

**Reply by skoruba** [reply]
Hello, you can use also a template project and you can override existing view/controller according to original path from UI package. You can take a look at razor class library how they have built the system of these things. 🙂

Thanks
